src/main/webapp/consentGroup/NewConsentGroup.js

```diff
@@ -81,17 +84,22 @@ class NewConsentGroup extends Component {
 
   submitNewConsentGroup = () => {
 
+    this.setState(prev => {
```

Comment:
for future use : we should plain _**this.setState({ submitError: false });**_ format when possible. No need to use "prev" approach here.